core/mapper/exprmapper/expression/expression.go

```diff
@@ -1,150 +1,38 @@
 package expression
 
 import (
+	"fmt"
+
 	"github.com/TIBCOSoftware/flogo-lib/core/mapper/exprmapper/expression/expr"
-	"github.com/TIBCOSoftware/flogo-lib/core/mapper/exprmapper/expression/function"
 	"github.com/TIBCOSoftware/flogo-lib/core/mapper/exprmapper/expression/gocc/lexer"
 	"github.com/TIBCOSoftware/flogo-lib/core/mapper/exprmapper/expression/gocc/parser"
-	"github.com/TIBCOSoftware/flogo-lib/core/data"
 	"github.com/TIBCOSoftware/flogo-lib/logger"
-	"github.com/TIBCOSoftware/flogo-lib/core/mapper/exprmapper/funcexprtype"
 )
 
 var log = logger.GetLogger("expression")
 
-const (
-	EXPRESSION = iota
-	TERNARY_EXPRESSION
-	FUNCTION
-	STRING
-)
-
-type functionExpression struct {
-	expr string
-}
-
-type weExpr struct {
-	expr string
-}
-
-func NewFunctionExpression(expression string) *functionExpression {
-	return &functionExpression{expr: expression}
-}
-
-func NewExpression(expression string) *weExpr {
-	return &weExpr{expr: expression}
-}
-
-func (f *functionExpression) Eval() ([]interface{}, error) {
-	function, err := f.GetFunction()
+func ParserExpression(exprString string) (expr.Expr, error) {
```

Comment:
Should this be ParseExpression?

Reply:
Yes. it should be a verb. will fix it now